Various non-normative HTML fixes

dr-shorthair opened the pull request:
fixes #143

dr-shorthair committed:
Various non-normative HTML fixes addressing #143

dr-shorthair committed:
'an' better than 'a' before OWL

dr-shorthair committed:
Merge branch 'gh-pages' into fix-html

# Conflicts:
#	ssn/index.html

dr-shorthair committed:
merge sosa-oms.ttl

dr-shorthair committed:
re-run 'Format document' in VSCode (indents and linefeeds)

dr-shorthair committed:
Three more update notes

dr-shorthair committed:
Fix internal hrefs

dr-shorthair committed:
Add missing terms to Overview A-Z

dr-shorthair committed:
Is OMS alignment normative here?

dr-shorthair committed:
Overview properties list A-Z

dr-shorthair committed:
linefeeds

dr-shorthair committed:
Remove duplicated section on OBOE alignment (old)

dr-shorthair commented:
Could one more person review so this can be merged?

ldesousa reviewed (changes requested):
@dr-shorthair This is a very large set of changes, since it is non-normative I didin't look into detail to the content, but a few points are worth considering:

1. There are many "... TODO ..." notes across the document. Is it on purpose? Eve if yes consider instead to open an issue pointing to the missing content.

2. It looks like some of the inner blue tables are missing cells (i.e. the background is not applied). Some examples below.

![Selection_121](https://github.com/w3c/sdw-sosa-ssn/assets/1137878/ebcc8789-e443-4aca-80e6-0c39ab36f011)
![Selection_122](https://github.com/w3c/sdw-sosa-ssn/assets/1137878/57cbb96e-d861-4f27-8ebb-c83f65061fb4)

3. There are many broken links throughout the document. In Brave (Chromium) the HTML render marks those with a red wavy underline. A few examples below.

![Selection_126](https://github.com/w3c/sdw-sosa-ssn/assets/1137878/d03f3a95-659a-49f9-9b41-1576e3f9423a)
![Selection_125](https://github.com/w3c/sdw-sosa-ssn/assets/1137878/0ded3a93-7ea3-4af2-81d6-02afe534208e)
![Selection_124](https://github.com/w3c/sdw-sosa-ssn/assets/1137878/5d990fc4-da24-4bda-952f-9dc4c53c2005)
![Selection_123](https://github.com/w3c/sdw-sosa-ssn/assets/1137878/7150994c-0659-4c7e-b9af-b069fd6acebb)

sgrellet committed:
passed the file to https://validator.w3.org/. Just fixed some of the 'Error' from the report. I hope didn't break anything

dr-shorthair committed:
1. minor change to non-normative header explanations
2. add pointer to [Hide SSN axioms] etc
3. Fix broken internal links

dr-shorthair committed:
Remove superfluous <section>
Fix property name (a very old bug!)

dr-shorthair commented:
Thank you @ldesousa 

1. The TODO comments are either
  - need to redraw figures - see #140 - but note that this will be deferred until the ontology is stable
  - need for examples for the Collection terms - see new issue #159
2. The inner blue tables show the axioms in the SSN graph. If you click the link `[Hide additional SSN axioms]` it will be hidden. I've added a note in the Modularization section drawing attention to this capability
3. I've fixed 7 broken links (internal cross-references). The red underlines associated with the show/hide buttons are an artefact of javascript and relative paths which should go away when the branch is merged. 

I also accepted the lint corrections from @sgrellet 

See https://raw.githack.com/w3c/sdw-sosa-ssn/fix-html/ssn/index.html#Specification for the revised state

dr-shorthair committed:
Merge branch 'gh-pages' into fix-html

# Conflicts:
#	ssn/index.html

dr-shorthair committed:
A-Z order in SOSA-OMS alignment

dr-shorthair committed:
Tidy up formatting of HFoI and hUFoI scope note

dr-shorthair committed:
Cleared all the bugs from https://validator.w3.org/nu/
